types/mongoose/mongoose-tests.ts

```diff
@@ -1215,6 +1215,10 @@ MongoModel.find({}).$where('indexOf("val") !== -1').exec(function (err, docs) {
 MongoModel.findById(999, function (err, doc) {
   doc.increment();
   doc.save(cb).then(cb).catch(cb);
+  doc.save({ validateBeforeSave: false }, cb).then(cb).catch(cb);
```

Comment:
@fabiob could you change this line and the one above to:
```js
doc.save(function(err, product, numAffected) {}).then(cb).catch(cb);
doc.save({ validateBeforesave: false }, function(err, product, numAffected) {
}).then(cb).catch(cb);
```
So that we check the callback has the right types?